<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="4" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-13T23:06:27 IST" name="Suite" finished-at="2024-09-13T23:06:36 IST" duration-ms="9272">
    <groups>
    </groups>
    <test started-at="2024-09-13T23:06:27 IST" name="Test" finished-at="2024-09-13T23:06:36 IST" duration-ms="9272">
      <class name="api.test.PetTest">
        <test-method is-config="true" signature="setUpData()[pri:0, instance:api.test.PetTest@7cbc3762]" started-at="2024-09-13T23:06:27 IST" name="setUpData" finished-at="2024-09-13T23:06:27 IST" duration-ms="229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpData -->
        <test-method signature="getPetTest()[pri:1, instance:api.test.PetTest@7cbc3762]" started-at="2024-09-13T23:06:27 IST" name="getPetTest" finished-at="2024-09-13T23:06:32 IST" duration-ms="4505" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetTest -->
        <test-method signature="PutPetTest()[pri:2, instance:api.test.PetTest@7cbc3762]" started-at="2024-09-13T23:06:32 IST" name="PutPetTest" finished-at="2024-09-13T23:06:35 IST" duration-ms="2970" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PutPetTest -->
        <test-method signature="DeletePetTest()[pri:3, instance:api.test.PetTest@7cbc3762]" started-at="2024-09-13T23:06:35 IST" name="DeletePetTest" finished-at="2024-09-13T23:06:36 IST" duration-ms="1291" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at api.test.PetTest.DeletePetTest(PetTest.java:114)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeletePetTest -->
      </class> <!-- api.test.PetTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
